third_party/miniz-2.1.0/miniz.c

```diff
@@ -6308,7 +6309,10 @@ mz_bool mz_zip_writer_add_mem_ex_v2(mz_zip_archive *pZip, const char *pArchive_n
 
 	if (!(level_and_flags & MZ_ZIP_FLAG_COMPRESSED_DATA))
 	{
-		uncomp_crc32 = (mz_uint32)mz_crc32(MZ_CRC32_INIT, (const mz_uint8 *)pBuf, buf_size);
+		// Cannot compute crc_32 with no payload
```

Comment:
I don't think "payload" is a concept here? I don't think you need a comment with the right variable names.